contracts/ITokenListManager.sol

```diff
@@ -1,7 +1,16 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.20;
 
-interface ITokenListManager {
+import "@openzeppelin/contracts/interfaces/IERC165.sol";
+
+interface ITokenListManager is IERC165 {
```

Comment:
Why inherit from `IERC165`?

Reply:
Forgot to check my code when I made some change. it's emoved right now.